app/Http/Routes/AdminRoutes.php

```diff
@@ -121,6 +121,21 @@ public function map(Registrar $router)
                 $router->delete('{incident_template}/delete', 'IncidentController@deleteTemplateAction');
             });
 
+            $router->group(['prefix' => 'subscribers', 'middleware' => 'app.subscribers'], function ($router) {
```

Comment:
Hmmm... Let me have a think on this. We need it to still work in the event that they just want to add people and auto-verify them.